pyro/infer/autoguide/initialization.py

```diff
@@ -54,10 +55,20 @@ def init_to_sample(site=None):
     return site["fn"].sample().detach()
 
 
-def init_to_median(site=None, num_samples=15):
+def init_to_median(
+    site=None,
+    num_samples=15,
+    *,
+    fallback: Optional[Callable] = init_to_feasible,
```

Comment:
I've added `fallback` kwargs to a few init strategies mainly to make it easier for users to catch unintended initialization.